src/main/scala/tile/BaseTile.scala

```diff
@@ -248,11 +249,9 @@ abstract class BaseTile private (val crossing: ClockCrossingType, q: Parameters)
 
   /** Node for the core to drive legacy "raw" instruction trace. */
   val traceSourceNode = BundleBridgeSource(() => Vec(traceRetireWidth, new TracedInstruction()))
-  /** Node to broadcast legacy "raw" instruction trace. */
-  val traceNexusNode = BundleBroadcast[Vec[TracedInstruction]]()
   /** Node for external consumers to source a legacy instruction trace from the core. */
   val traceNode: BundleBridgeOutwardNode[Vec[TracedInstruction]] =
-    BundleBridgeNameNode("trace") :*= traceNexusNode := traceSourceNode
+    BundleBridgeNameNode("trace") := traceSourceNode
```

Comment:
So this moves `traceNexusNode` to the PRCI layer so it can be blocked during reset.  There's some technical debt to apply the same treatment to `traceCoreNexusNode` (below).  This bundle doesn't have a valid signal.  Either we will have to add a `valid` to `TraceCoreInterface` (bending the definition of this interface from the RISC-V Trace Spec) or add another form of `BlockDuringReset` specifically for this (which would zero out `iretire` and `itype`).

Reply:
Thanks for the feedback, I was wondering a bit about whether that bundle needed to be blocked too. There's almost certainly a way to have each Bundle subclass define it's own `blockMe` function (name TBD) that returns a copy of itself but with certain signals tied low. You still have to make one NexusNode per type of bundle bridge, but it's at least very generic code to ensure that the blocking occurs.